### PR TITLE
Update main.cpp

### PR DESCRIPTION
**Pull request by ThoYuHan**

### CODE REVIEW

**Review by aidasakinah — CHANGES_REQUESTED** [review]
where did lines 6-9 go ?? I need them for my functions

File: main.cpp
Comment (aidasakinah) [inline main.cpp:1]: yuhan, where did all the previous functions go??

**Comment by ThoYuHan** [comment]
> where did lines 6-9 go ?? I need them for my functions

i didnt combine your code yet. it is just my part

**Review by aidasakinah — CHANGES_REQUESTED** [review]
why is the code you've submitted the old version? It isn't the one I gave this morning

**Comment by aidasakinah** [comment]
> > where did lines 6-9 go ?? I need them for my functions
> 
> i didnt combine your code yet. it is just my part

For me to combine the code into main, you have to merge it with the old files or else it's deleted

File: main.cpp
Comment (ThoYuHan) [reply main.cpp:1]: > yuhan, where did all the previous functions go??

is all there. i didnt delete any function

**Comment by ThoYuHan** [comment]
> > > where did lines 6-9 go ?? I need them for my functions
> > 
> > 
> > i didnt combine your code yet. it is just my part
> 
> For me to combine the code into main, you have to merge it with the old files or else it's deleted

you mean i need to combine yours then update?

**Pull request merged**